workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandResponseHandler.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core.Commands;
+using Unity.Entities;
+using Entity = Unity.Entities.Entity;
+
+#region Diagnostic control
+
+// ReSharper disable UnusedMember.Local
+
+#endregion
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommandsRequirables
+    {
+        [InjectableId(InjectableType.WorldCommandResponseHandler, InjectableId.NullComponentId)]
+        [InjectionCondition(InjectionCondition.RequireNothing)]
+        public class WorldCommandResponseHandler : IInjectable
+        {
+            private readonly ILogDispatcher logDispatcher;
+
+            private readonly List<Action<WorldCommands.ReserveEntityIds.ReceivedResponse>> reserveEntityIdsDelegates;
+            private readonly List<Action<WorldCommands.CreateEntity.ReceivedResponse>> createEntityDelegates;
+            private readonly List<Action<WorldCommands.DeleteEntity.ReceivedResponse>> deleteEntityDelegates;
+
+            private WorldCommandResponseHandler(ILogDispatcher logDispatcher)
+            {
+                this.logDispatcher = logDispatcher;
+                reserveEntityIdsDelegates = new List<Action<WorldCommands.ReserveEntityIds.ReceivedResponse>>();
+                createEntityDelegates = new List<Action<WorldCommands.CreateEntity.ReceivedResponse>>();
+                deleteEntityDelegates = new List<Action<WorldCommands.DeleteEntity.ReceivedResponse>>();
+            }
+
+            public event Action<WorldCommands.ReserveEntityIds.ReceivedResponse> OnReserveEntityIdsResponse
+            {
+                add => reserveEntityIdsDelegates.Add(value);
```

Comment:
Now that #288 has been merged, could you wrap this in
```
if (!VerifyNotDisposed())
{
  return;
}
``` 
blocks?

Reply:
aye